### PR TITLE
update solana

### PR DESCRIPTION
**Pull request by jacobcreech**
Added crawled repos between December 1, 2022 and now. Roughly 1866 repos added.

### CODE REVIEW

File: data/ecosystems/s/solana.toml [hunk 1]
```diff
@@ -8727,3 +8727,5602 @@ url = "https://github.com/ZYJLiu/token-creation-frontend"
...
+url = "https://github.com/kumailnaqvi354/Solana-Todo"
+
+[[repo]]
+url = "https://github.com/gcunruh/tipsea-solana"
+
+[[repo]]
+url = "https://github.com/shumshersubashgautam/instasolanaweb3.0"
+
+[[repo]]
+url = "https://github.com/bilira-org/tender"
```
Comment (eherrerosj) [inline data/ecosystems/s/solana.toml:13340]: can you create a separate ecosystem for bilira?

File: data/ecosystems/s/solana.toml [hunk 2]
```diff
@@ -8727,3 +8727,5602 @@ url = "https://github.com/ZYJLiu/token-creation-frontend"
...
+url = "https://github.com/BonketyBonk/bonkety-frontend"
+
+[[repo]]
+url = "https://github.com/web3-payments/cross-pay-solana-smart-contracts"
+
+[[repo]]
+url = "https://github.com/nathanramli/solana-crowdfunding"
+
+[[repo]]
+url = "https://github.com/magiclabs/example-solana"
```
Comment (eherrerosj) [inline data/ecosystems/s/solana.toml:10382]: can remove, I added Magic ecosystem to Solana

File: data/ecosystems/s/solana.toml [hunk 3]
```diff
@@ -8727,3 +8727,5602 @@ url = "https://github.com/ZYJLiu/token-creation-frontend"
...
+url = "https://github.com/honey-labs/honey-sdk"
+
+[[repo]]
+url = "https://github.com/dao-xyz/shorbit"
+
+[[repo]]
+url = "https://github.com/labeleven-dev/bettercallsol"
+
+[[repo]]
+url = "https://github.com/CFXLabsInc/cfx-solana-contexts"
```
Comment (eherrerosj) [inline data/ecosystems/s/solana.toml:8891]: Can you please create a separate ecosystem for CFX Labs and remove this line?

File: data/ecosystems/s/solana.toml [hunk 4]
```diff
@@ -8727,3 +8727,5602 @@ url = "https://github.com/ZYJLiu/token-creation-frontend"
...
+url = "https://github.com/Gari-Dev-DAO/Burn_them_all"
+
+[[repo]]
+url = "https://github.com/void-ness/Soul-connects-program"
+
+[[repo]]
+url = "https://github.com/ayorkshireworrall/token-sale"
+
+[[repo]]
+url = "https://github.com/solpipe/solana-firehose"
```
Comment (eherrerosj) [inline data/ecosystems/s/solana.toml:11780]: Can you create a separate ecosystem for Solpipe and add its org and remove this line?

File: data/ecosystems/s/solana.toml [hunk 5]
```diff
@@ -8727,3 +8727,5602 @@ url = "https://github.com/ZYJLiu/token-creation-frontend"
...
+url = "https://github.com/eyusufatik/learning-rust-and-solana"
+
+[[repo]]
+url = "https://github.com/thinkrapido/RoboSwapProgram"
+
+[[repo]]
+url = "https://github.com/Altar12/on-chain-tic-tac-toe-stake"
+
+[[repo]]
+url = "https://github.com/thirdweb-example/next-javascript-solana-starter"
```
Comment (eherrerosj) [inline data/ecosystems/s/solana.toml:13400]: can you create an ecosystem for Thirdweb Templates, remove the repos in here that belong to it and add the ecosystem as a sub ecosytems to solana?

**Review by eherrerosj — CHANGES_REQUESTED** [review]
hey @jacobcreech, thanks for the PR! Overall it looks good. I have added a few change requests so we can isolate certain sub-ecosystems under Solana. We should also remove a few duplicates and repos that are not primary to Solana

File: data/ecosystems/s/solana.toml [hunk 6]
```diff
@@ -8727,3 +8727,5602 @@ url = "https://github.com/ZYJLiu/token-creation-frontend"
...
+url = "https://github.com/pinocchio338/boss_meta"
+
+[[repo]]
+url = "https://github.com/mateolafalce/Block3_data_store"
+
+[[repo]]
+url = "https://github.com/mateolafalce/Decenwser-rust"
+
+[[repo]]
+url = "https://github.com/cinojosa0705/xnft-template-stacc"
```
[the remark this answers is not in the document]
Comment (jacobcreech) [reply data/ecosystems/s/solana.toml:10445]: xnfts started on Solana, and we can see the repo currently is using [@solana/web3.js](https://github.com/cinojosa0705/xnft-template-stacc/blob/master/package.json#L19)

File: data/ecosystems/s/solana.toml (hunk 5 above)
Comment (jacobcreech) [reply data/ecosystems/s/solana.toml:13400]: Thirdweb does a lot more than Solana. Does it make sense to make it a subecosystem to Solana?

Comment (eherrerosj) [reply data/ecosystems/s/solana.toml:13400]: Good observation. You can add it as a sub ecosystem of Solana and I will add it as a sub-ecosystem of the other compatible ecosystems later on

File: data/ecosystems/s/solana.toml (hunk 6 above)
[the remark this answers is not in the document]
Comment (eherrerosj) [reply data/ecosystems/s/solana.toml:10445]: cool, just saw it, thanks

**Comment by jacobcreech** [comment]
@eherrerosj Made all the updates requested!

File: data/ecosystems/t/thirdweb.toml [hunk 7]
```diff
@@ -3,7 +3,7 @@ title = "thirdweb"
 
 sub_ecosystems = []
 
-github_organizations = ["https://github.com/thirdweb-dev"]
+github_organizations = ["https://github.com/thirdweb-dev", "thirdweb-example"]
```
Comment (eherrerosj) [inline data/ecosystems/t/thirdweb.toml:6]: the org should be a URL, like the first element in there

**Review by eherrerosj — CHANGES_REQUESTED** [review]
good progress! almost there, just left one last comment. Btw, just FYI, we automatically pick all repos out of github orgs, then we can manually black list in our infra

**Review by eherrerosj — APPROVED** [review]
Looks good now! Thank you @jacobcreech

**Pull request merged**